qemu/tests/cfg/numa_opts.cfg

```diff
@@ -92,3 +92,18 @@
               extra_params += " -numa node,mem=2560,cpus=5,nodeid=2"
               extra_params += " -numa node,mem=512,cpus=3-4,nodeid=0"
               extra_params += " -numa node,mem=1024,cpus=0-2,nodeid=1"
+    - nodes.128:
+        type = numa_maxnodes
+        numa_nodes = 128
+        mem_fixed = 4G
+        use_mem = no
+        node_size = 32M
```

Comment:
Better define `start_vm = no`, otherwise I think the vm is processed twice.

Reply:
It has been defined in L5.